test/image/mocks/axes_exponentbase_linear.json

```diff
@@ -0,0 +1,217 @@
+{
+    "data": [
+        {
+            "x": [
+                1,
+                2,
+                3
+            ],
+            "y": [
+                4096,
+                8192,
+                16384
+            ],
+            "name": "yaxis1 data",
```

Comment:
minor point: would you mind changing the trace `name` to their axis `exponentbase` value? This would make the image easier to grasp.